.github/workflows/lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Linting Checks
```

Comment:
Replace lint with tests check

Reply:
are you suggesting to run test as well as part of this job

Reply:
Yes, just tests. No lint checks.

Reply:
done